extensions/machine-learning/src/common/apiWrapper.ts

```diff
@@ -137,4 +138,16 @@ export class ApiWrapper {
 	public registerWidget(widgetId: string, handler: (view: azdata.ModelView) => void): void {
 		azdata.ui.registerModelViewProvider(widgetId, handler);
 	}
+
+	private azurecoreApi: azurecore.IExtension | undefined;
+
+	public async getAzurecoreApi(): Promise<azurecore.IExtension> {
+		if (!this.azurecoreApi) {
```

Comment:
this won't wont return error for when user doesn't have access to tenant like the previous one did?

Reply:
What do you mean? This method is just getting the azurecore API through which you call getSubscriptions/getResourceGroups. Those calls are unchanged functionality-wise.

Reply:
hmm ok my mistake, I thought those were in different extension and you're using new APIs.

Reply:
They are in a different extension (azurecore)